src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,246 @@
+        </v-col>
+        <v-col cols="7" class="pt-0 pl-0 info-text" v-else>
+          This company has Pre-existing Company Provisions.
+        </v-col>
+        <v-col mt-n2 cols="2" class="pt-0 align-right" v-if="!hasProvisionsRemovedPropsChanged">
+          <v-btn id="correct-company-provisions" text color="primary" @click="isEditing = true">
+            <v-icon small>mdi-pencil</v-icon>
+            <span>{{ editLabel }}</span>
+          </v-btn>
+        </v-col>
+        <v-col pt-0 mt-n2 cols="2" class="align-right" v-else>
+          <v-btn
+            id="undo-company-provisions"
+            text
+            color="primary"
+            class="undo-company-provisions"
+            @click="resetCompanyProvisions"
+          >
+            <v-icon small>mdi-undo</v-icon>
+            <span>Undo</span>
+          </v-btn>
+
+          <!-- More Actions Menu -->
+          <span class="more-actions">
+            <v-menu offset-y>
+              <template v-slot:activator="{ on }">
+                <v-btn text small color="primary" class="more-actions-btn" v-on="on">
+                  <v-icon>mdi-menu-down</v-icon>
+                </v-btn>
+              </template>
+              <v-list>
+                <v-list-item class="actions-dropdown_item" @click="isEditing = true">
+                  <v-list-item-subtitle>
+                    <v-icon small>mdi-pencil</v-icon>
+                    <span class="ml-1">{{ editLabel }}</span>
+                  </v-list-item-subtitle>
+                </v-list-item>
+              </v-list>
+            </v-menu>
+          </span>
+        </v-col>
+      </v-row>
+    </v-container>
+    <v-container pa-0 ma-0 v-else>
+      <v-row no-gutters>
+        <v-col class="pr-0 pl-0 pt-0" cols="3">
+          <label class="font-weight-bold">Pre-existing Company Provisions</label>
+        </v-col>
+        <v-col class="pt-0 pl-0" cols="8">
+          <v-row no-gutters class="pl-1 pa-0 ma-0">
+            <p class="info-text mb-0">
+              Complete this item only if the company has resolved that none of the Pre-existing Company Provisions
+              are to apply to this company (refer to Part 17 and Table 3 of the Regulation under the Business
+              Corporations Act).
+            </p>
+            <div class="d-flex align-start">
+                <v-checkbox ma-0 pa-0
+                  id="cp-checkbox"
+                  :class="{ 'invalid': isInvalid }"
+                  ref="checkboxRef"
+                  v-model="draftProvisionsRemoved"
+                />
+                <span class="info-text checkbox-label" :class="{ 'invalid': isInvalid }">
+                  The company has resolved that none of the Pre-existing Company Provisions are to apply to this
+                  company.
+                </span>
+            </div>
+          </v-row>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-container pt-10>
+          <v-row xs12>
```

Comment:
I don't think row elements take this attribute. I think it's only for col elements.

Reply:
👍